spec.html

```diff
@@ -24900,6 +24944,14 @@ <h1>Static Semantics: ExportedBindings</h1>
         <emu-alg>
           1. Return a List containing the StringValue of the first |IdentifierName|.
         </emu-alg>
+        <emu-grammar>ExportSpecifier : ModuleExportName `as` ModuleExportName</emu-grammar>
+        <emu-alg>
+          1. Return a List containing the StringValue of the first |ModuleExportName|.
+        </emu-alg>
+        <emu-grammar>ExportSpecifier : ModuleExportName</emu-grammar>
+        <emu-alg>
+          1. Return a List containing the StringValue of |ModuleExportName|.
+        </emu-alg>
```

Comment:
This should also be sorted:
```suggestion
        <emu-grammar>
          ExportSpecifier :
            ModuleExportName
            ModuleExportName `as` IdentifierName
        </emu-grammar>
        <emu-alg>
          1. Return a List containing the StringValue of |ModuleExportName|.
        </emu-alg>
        <emu-grammar>ExportSpecifier : ModuleExportName `as` ModuleExportName</emu-grammar>
        <emu-alg>
          1. Return a List containing the StringValue of the first |ModuleExportName|.
        </emu-alg>
```